debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/ObjectFieldInfo.java

```diff
@@ -407,8 +407,8 @@ boolean isInstanceFieldBackfillEligible() {
 								if (valueTypeHelper.classRequires4BytePrePadding(fieldClass)) {
 									instanceSize = instanceSize.sub(U32.SIZEOF);
 								}
-								forceDoubleAlignment = modifiers.allBitsIn(J9JavaAccessFlags.J9AccVolatile)
-										&& instanceSize.eq(doubleSize);
+								forceDoubleAlignment = (modifiers.allBitsIn(J9JavaAccessFlags.J9AccVolatile) || valueTypeHelper.isRomClassAtomic(fieldClass.romClass()))
+														&& instanceSize.eq(doubleSize);
```

Comment:
Please restore indentation of this line.

Reply:
Fixed.